main.tf

```diff
@@ -45,6 +45,17 @@ resource "ibm_kms_instance_policies" "key_protect_instance_policies" {
   }
 }
 
+locals {
```

Comment:
Is there a provider issue for this workaround? If so please paste it as a code comment here

Reply:
Just opened https://github.com/IBM-Cloud/terraform-provider-ibm/issues/5163

Reply:
added a link to the provider issue in the local, creating an issue on our side to track the provider issue/remove this workaround once output issue is fixed